data/base-hunting.yaml

```diff
@@ -631,12 +631,23 @@ hunting_zones:
   - 1554
   # https://elanthipedia.play.net/Rock_Troll_(1)                           60-73
   rock_trolls:
-  - 8946
```

Comment:
Is this the room with the unmovable rock? Can you please comment it out if so and and comment about why it was removed. This is just so that we can remember it within the codebase and nudge others from re-adding it.

Reply:
Ah missed my chance :P but feel free to address the comment in a future PR :)

Reply:
8955 is the room with the unmoveable rock.  8946 is the first room in the zone across the barrier and group members will be dumped there if you are not careful.  i took it out for this reason.